tool/appveyor.ps1

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2014, Google Inc. Please see the AUTHORS file for details.
+# All rights reserved. Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+param([string]$action)
+
+function throw_if_process_failed {
+    param([string]$message)
```

Comment:
Not sure about an established convention, but I'd say 4 spaces is common. I've seen tabs too, though.

BTW, I'm not following PowerShell idiosyncrasies to the letter... For example, I should use capitals instead of underscores, etc.

@vors Sorry to bother you. Re. whitespace, do you know if there's a style guide or anything like that we could refer to?